Creating data table

Krittapas2004 opened the pull request:
I've fixed the missing .env file and resolved the server startup error.

Commit:
Fixing the data table

Commit:
Fixing the import and export of the data

Krittapas2004 committed:
Fixing an api not posting in postman

Krittapas2004 committed:
trying to add env file manully

Krittapas2004 committed:
Fixed the api not posting the data onto postman

Navheen0508 reviewed (changes requested):
There is an issue with the way that the table has been migrated as the error messaging is displaying that the Agent Data table doesn't exist in the database when trying to create the data of a new agent. 

![image](https://github.com/BIT-Studio-3/2024-rest-api-dark-mode/assets/131924590/b23e56c9-40bd-4beb-a261-a1872b0cf14b)

Other KRUD related requests do not work as well - this needs to be fixed before this branch can be merged into main. I am unable to identify the problem myself as after reviewing the code for your KRUD requests in your controllers and routes file, everything seems to be correct. The error suggests that it's something to do with the migration so I'd suggest looking into that as a starting point.

Krittapas2004 committed:
fix(Contract-API): Error with the migration

Fix API creation issue due to edited migration file without table migration.

Krittapas2004 commented:
### Fixed Error
The issue has been resolved—I fixed the problem, which turned out to be related to a database migration. Once I removed the old migration, everything ran smoothly.

If you encounter the issue again, consider deleting the previous migration file and running the migration process anew. It could be due to a Prisma update.

Krittapas2004 committed:
Deleting the prima file and moving it to staging branch
